libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -32,9 +33,7 @@ export class BookSearchComponent implements OnInit {
   }
 
   ngOnInit(): void {
-    this.store.select(getAllBooks).subscribe(books => {
-      this.books = books;
-    });
+    this.books$ = this.store.select(getAllBooks);
   }
 
   formatDate(date: void | string) {
```

Comment:
Have you given any thought to this method?

Reply:
Removed this method and used DatePipe in html.